src/ontology/mondo-ingest.Makefile

```diff
@@ -128,6 +128,24 @@ $(COMPONENTSDIR)/icd10who.owl: $(TMPDIR)/icd10who_relevant_signature.txt | compo
 		remove -T config/properties.txt --select complement --select properties --trim true \
 		annotate --ontology-iri $(URIBASE)/mondo/sources/icd10who.owl --version-iri $(URIBASE)/mondo/sources/$(TODAY)/icd10who.owl -o $@; fi
 
+.PHONY: component-download-gard.owl
+component-download-gard.owl: | $(TMPDIR)
+	if [ $(MIR) = true ] && [ $(COMP) = true ]; then $(ROBOT) merge -I https://github.com/monarch-initiative/gard/releases/latest/download/gard.owl \
+	annotate --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) -o $(TMPDIR)/$@.owl; fi
+
+$(COMPONENTSDIR)/gard.owl: $(TMPDIR)/gard_relevant_signature.txt | component-download-gard.owl
+	if [ $(COMP) = true ]; then $(ROBOT) remove -i $(TMPDIR)/component-download-gard.owl.owl --select imports \
+		rename --mappings config/property-map-1.sssom.tsv --allow-missing-entities true \
+		rename --mappings config/property-map-2.sssom.tsv --allow-missing-entities true \
+		remove -T $(TMPDIR)/gard_relevant_signature.txt --select complement --select "classes individuals" --trim false \
+		remove -T config/remove.txt --axioms equivalent \
+		query \
+			--update ../sparql/fix-labels-with-brackets.ru \
+			--update ../sparql/fix_hgnc_mappings.ru \
+			--update ../sparql/fix_complex_reification.ru \
```

Comment:
There seem to be many unnecessary steps here. Only add what is really necessary into the preprocessing pipeline.